ent/src/yb/tools/yb-admin_client_ent.cc

```diff
@@ -146,7 +147,8 @@ Status ClusterAdminClient::ListSnapshots(const ListSnapshotsFlags& flags) {
           HybridTimeToString(HybridTime::FromPB(entry.previous_snapshot_hybrid_time())),
           &json_snapshot, &document.GetAllocator());
     } else {
-      cout << SnapshotIdToString(snapshot.id()) << kColumnSep << snapshot.entry().state() << endl;
+      cout << SnapshotIdToString(snapshot.id()) << kColumnSep << snapshot.entry().state() << 
+        kColumnSep << HybridTimeToString(HybridTime::FromPB(snapshot.entry().snapshot_hybrid_time())) << endl;
```

Comment:
For consistency, we generally align the `couts` like here - https://github.com/yugabyte/yugabyte-db/blob/273994915af69d5cc5086779d25a196ed6539f8d/src/yb/tools/yb-admin_client.cc#L1062